src/tools/convert/topology_helper.py

```diff
@@ -58,7 +58,12 @@ def FillFileBuffer(fname, filebuffer):
     for line in f:
 	if "include" in line and not line[0]==';':
 	    name=(line.split()[1]).strip('\"')
-	    FillFileBuffer(name, filebuffer)
+            try: 
```

Comment:
No, but the git diff displays wrong because there's a weird mix of spaces and tabs in that py file. Will clean it up a bit now (at least the part I touched!)